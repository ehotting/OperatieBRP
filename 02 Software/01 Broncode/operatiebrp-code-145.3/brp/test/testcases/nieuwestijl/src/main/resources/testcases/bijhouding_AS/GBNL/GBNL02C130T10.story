Meta:
@status                 Klaar
@usecase                UCS-BY.0.AS

Narrative: R2455 Er moet precies één OUWKIG geregistreerd worden

Scenario: Er wordt geen OUWKIG geregistreerd
          LT: GBNL02C130T10

Given alle personen zijn verwijderd

!-- Geboorte van het kind zonder OUWKIG
When voer een bijhouding uit GBNL02C130T10.xml namens partij 'Gemeente BRP 1'

!-- XSD-error
Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding_AS/GBNL/expected/GBNL02C130T10.txt