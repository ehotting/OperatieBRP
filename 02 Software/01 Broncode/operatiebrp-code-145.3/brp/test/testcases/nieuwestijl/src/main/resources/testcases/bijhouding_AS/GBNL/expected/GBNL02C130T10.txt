cvc-complex-type.2.4.b: The content of element 'brp:betrokkenheden' is not complete. One of '{"http://www.bzk.nl/brp/brp0200":ouder}' is expected.